src/components/developed-by/developed-by.scss

```diff
@@ -0,0 +1,41 @@
+@import "~bulma/sass/utilities/_all.sass";
```

Comment:
Bulma files will be automatically imported as global styles in head, so we can remove that import.

Reply:
this is required because the build throws errors otherwise
Unless there is some extra configuration that needs to be done

![image](https://user-images.githubusercontent.com/6185049/76743921-10949500-677c-11ea-9ec5-3526f466f181.png)